src/lib.rs

```diff
@@ -3514,8 +3520,13 @@ impl Build {
                     "x86_64-unknown-netbsd" => Some("x86_64--netbsd"),
                     _ => None,
```

Comment:
I think all of this is unnecessary; we should just use:
```rust
"x86_64-unknown-linux-musl" => self.find_working_gnu_prefix(&["x86_64-linux-musl", "musl"]),
```

Reply:
Thanks, I will open another PR for this (since it's easier to start from scratch)